.github/workflows/velox_be.yml

```diff
@@ -168,3 +168,51 @@ jobs:
         if: ${{ always() }}
         run: |
           docker stop velox-backend-ubuntu2204-test-$GITHUB_RUN_ID || true
+  
+  velox-backend-centos8-test:
+    runs-on: velox-self-hosted
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup docker container
+        run: |
+          EXTRA_DOCKER_OPTIONS="--name velox-backend-centos8-test-$GITHUB_RUN_ID -e NUM_THREADS=30 --detach" \
+          NON_INTERACTIVE=ON \
+          MOUNT_MAVEN_CACHE=OFF \
+          OS_IMAGE=centos:8 \
+          tools/gluten-te/centos/cbash.sh sleep 14400
+      - name: Setup maven cache
+        run: |
+          docker cp ~/.m2/repository velox-backend-centos8-test-$GITHUB_RUN_ID:/root/.m2/
+      - name: Build Gluten Velox-backend third party
+        run: |
+          docker exec velox-backend-centos8-test-$GITHUB_RUN_ID bash -c '
+          source /opt/rh/gcc-toolset-9/enable && \
+          cd /opt/gluten/ep/build-arrow/src && \
+          ./get_arrow.sh  && \
+          ./build_arrow_for_velox.sh && \
+          cd /opt/gluten/ep/build-velox/src
+          ./get_velox.sh && \
+          ./build_velox.sh'
+      - name: Build Gluten CPP library
+        run: |
+          docker exec velox-backend-centos8-test-$GITHUB_RUN_ID bash -c '
+          source /opt/rh/gcc-toolset-9/enable && \
+          cd /opt/gluten/cpp && \
+          mkdir build && cd build && cmake .. && make -j'
```

Comment:
Now we repeatedly build protobuf in CI, both build velox and build cpp library, maybe we can set `BUILD_PROTOBUF` default to `OFF`, https://github.com/oap-project/gluten/blob/main/cpp/CMakeLists.txt#L45

Reply:
I will check it.